test/api/v3/integration/user/POST-user_class_cast_spellId.test.js

```diff
@@ -187,6 +187,30 @@ describe('POST /user/class/cast/:spellId', () => {
     expect(group.chat[0].uuid).to.equal('system');
   });
 
+  it('Ethereal Surge does not recover mp of other mages', async () => {
+    let group = await createAndPopulateGroup({
+      groupDetails: { type: 'party', privacy: 'private' },
+      members: 4,
+    });
+    await group.groupLeader.update({'stats.mp': 200, 'stats.class': 'wizard', 'stats.lvl': 20});
```

Comment:
this 5 updates can be made in parallel with `Promise.all(arrayofUpdates)` which will help with speed, same for the sync calls below. Is the sleep necessary?

Reply:
I saw sleep on a lot of tests in that script, so I considered it was necessary.
Sure I can remove it.